public/js/build/BuildRecipeFunctionFactory.js

```diff
@@ -8,13 +8,19 @@ class BuildRecipeFunctionFactory {
    */
   createAddIngredientFunction(ingredient_id, ingredient_name, brvc) {
     return function () {
-
       if (ingredient_id in brvc.recipe_ingredients) {
-        alert(`Ingredient ${ingredient_name} (ID ${ingredient_id}) is already in the recipe.`);
+        // show notification that adding ingredient was not successful
+        let notifDiv = document.getElementById("alert-div");
+        let notif = document.createElement("div");
+        notif.textContent = `${ingredient_name} (ID ${ingredient_id}) is already in the recipe.`;
+        notif.classList.add("alert", "add-error");
+        notifDiv.prepend(notif);
+        setTimeout(function() { notif.remove() }, 2000);
```

Comment:
Can we increase the timeout to 10 or 20 seconds?

Reply:
I increased the timeout to 10 seconds.